packages/lib/src/components/ThreeDS2/ThreeDS2.scss

```diff
@@ -6,6 +6,8 @@
     position: relative;
     overflow: hidden;
     width: 100%;
+    min-height: 400px;
```

Comment:
Is there any reason to force this to `400px`?

Reply:
Yes - 'cos if the merchant has never defined a height on any of the container elements the challenge window will not be visible